core/src/test/kotlin/org/evomaster/core/search/service/StatisticsTest.kt

```diff
@@ -0,0 +1,31 @@
+package org.evomaster.core.search.service
+
+import org.junit.Test
+import org.junit.jupiter.api.Assertions.assertEquals
+
+class StatisticsTest {
+
+    @Test
+    fun testMongoHeuristicsAverage() {
+        val statistics = Statistics()
+        statistics.reportNumberOfEvaluatedDocumentsForMongoHeuristic(10)
+        statistics.reportNumberOfEvaluatedDocumentsForMongoHeuristic(20)
+        statistics.reportNumberOfEvaluatedDocumentsForMongoHeuristic(30)
+
+        assertEquals(3, statistics.mongoHeuristicsEvaluationCount)
```

Comment:
i see... still put those value as `private` and then declare a `public get`, eg something like:
```
private mongoHeuristicsEvaluationCount : Int
      public get
```

Reply:
ok